packages/babel-plugin-proposal-logical-assignment-operators/test/fixtures/logical-assignment/anonymous-functions-exec/options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
+    "proposal-logical-assignment-operators",
+    "proposal-nullish-coalescing-operator"
```

Comment:
Note for reviewers: the "exec" and "transform" tests are in different folders because they have different configs.